README.md

```diff
@@ -56,7 +52,7 @@ available. This project uses autotools for the build process, allowing to
 have common build steps across different platforms.
 Only the prerequisites differ and they are described in this section.
 
-libusbmuxd requires [libplist](https://github.com/libimobiledevice/libplist) and [libimobiledevice-glue](https://github.com/libimobiledevice/libimobiledevice-glue). On Linux, it also requires [usbmuxd](https://github.com/libimobiledevice/usbmuxd) installed on the system, while macOS has its own copy and on Windows AppleMobileDeviceSupport package provides it.
+libusbmuxd requires [libplist](https://github.com/libimobiledevice/libplist) and [libimobiledevice-glue](https://github.com/libimobiledevice/libimobiledevice-glue). On Linux, it also requires [usbmuxd](https://github.com/libimobiledevice/usbmuxd) installed on the system, while macOS has its own copy and on Windows [AppleMobileDeviceSupport](TODO link) package provides it.
```

Comment:
I wasn't sure whether this is linking to `AppleMobileDeviceSupport.msi` (the installer) or `AppleMobileDeviceService` (the service provided by "it" and/or iTunes). Can anyone clarify so that we can link it appropriately?